data/IL/incentives.json

```diff
@@ -0,0 +1,1110 @@
+    "payment_methods": [
+      "account_credit"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "il_jo-CarrollEnergyCooperative_2023EnergyEfficiencyIncentives",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 25
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$25 bill credit for ENERGY STAR clothes dryer."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "IL-32",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy-company",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.5,
+      "maximum": 600
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $600 rebate for eligible air-source heat pumps, depending on unit efficiency, capped at 50% installed cost."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "IL-33",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy-company",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.5,
+      "maximum": 600
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $600 rebate for eligible ductless mini-split heat pumps, depending on unit efficiency, capped at 50% installed cost."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "IL-34",
+    "authority_type": "utility",
+    "authority": "il-mid-american-energy-company",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "il_midAmericanEnergyCompany_residentialInstantRebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.5,
+      "maximum": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $75 rebate for ENERGY STAR® Smart Thermostats, not to exceed 50% of purchase price."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "IL-35",
+    "authority_type": "state",
+    "authority": "il-state-of-illinois",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "weatherization",
+    "program": "il_stateOfIllinois_homeWeatherization",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 15000,
+      "maximum": 15000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $15,000 for energy-related weatherization and repair work."
+    },
+    "low_income": "il-state-of-illinois"
```

Comment:
Does it not work if you put `default` here?

Reply:
There's a thread on this here: https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1708549761480869

We can discuss on that thread, but switching these to default if there is only one program is fine for now.

Reply:
I couldn't actually get this to load the default without code changes [here](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/scripts/lib/spreadsheet-standardizer.ts#L159-L172). I couldn't find the logic elsewhere that would fallback to the default even though it seems like that happens elsewhere like NY. I could use some guidance unless we are okay moving forward with the change I suggested in the thread.

Reply:
I guess the problem is that the script we've been running `node build/scripts/incentive-spreadsheet-to-json.js` will clobber any manual edits we make to this file. So, even if I manually set this to `default` it gets overwritten on the next run. Perhaps that isn't a big deal now that this is generated?